### PR TITLE
Publish v14.20.0. [release]

### PR DESCRIPTION
**Pull request by schinery**
Creating 14.20.0 Docker files using `./shared/release.sh 14.20.0=lts`

### CODE REVIEW

File: ALIASES [hunk 1]
```diff
@@ -1,2 +1,2 @@
 current=18.4.0
-lts=16.16.0
+lts=14.20.0
```
Comment (schinery) [inline ALIASES:2]: Hmmm am wondering if this should be reverted?

**Comment by schinery** [comment]
@felicianotech @JalexChen is it possible to get this looked at? 🙏🏻

**Review by BytesGuy — APPROVED** [review]

**Comment by schinery** [comment]
Is this good to merge?

**Comment by JalexChen** [comment]
> 

Hi @schinery - this is getting merged right now. We ended up re-implementing vanilla node instead of nvm.

**Comment by schinery** [comment]
> > 
> 
> Hi @schinery - this is getting merged right now. We ended up re-implementing vanilla node instead of nvm.

@JalexChen do you need me to do anything for that in this PR?

**Comment by schinery** [comment]
Thanks @JalexChen 👍🏻

**Pull request merged**